README.md

```diff
@@ -22,20 +11,44 @@ The lesson outline and rough breakdown of topics by lesson writer is in
 [lesson-outline.md](lesson-outline.md). The topics there will be initially generated by the lesson
 writer, and then reviewed by the rest of the group once complete.
 
+## Using this material
+
+1. Follow the instructions found in the [Software Carpentry example lesson source](https://github.com/carpentries/lesson-example/)
+   to create a repository for your lesson.
+2. Edit [_config.yml](_config.yml) to modify the configuration options for the HPC system you
+   will be using in the section titled `Workshop specific values`. These options set such things
+   as the address of the host to login to, definitions of the command prompt, scheduler names.
+   The default is the setup for the Graham Compute Canada cluster hosted at the University
+   of Waterloo which uses the SLURM scheduler. Other examples can be found in the
+   `_extras/snippets_library/` directory.
+3. Create the required host-specific code snippets in subdirectories in
+   [_includes/snippets](_includes/snippets). These snippets provide inputs and outputs that 
+   are host-specific and that are included automatically when the lesson website is built.
+   1. Code snippets are in files named `snippet_name.snip` and are included automatically
+      when the lesson is built. For example, if the `snippet_name` was `login_output`,
+      then the snippet file would be called `login_output.snip`.
+   2. Code snippets are placed in subdirectories that are named according to the episode they
+      appear in. For example, if the snippet is for episode 01, then it will be in a 
+      subdirectory called `01`.
+   3. The default is the setup for the Graham Compute Canada cluster hosted at the University
+      of Waterloo which uses the SLURM scheduler. Other examples can be found in the
+      `_extras/snippets_library/` directory.
+
```

Comment:
I think `3.` should be replaced by an example on how the snippet is included in the `_episodes` markdown. I think point 3. is more a comment than an entry in a numbered list.

Reply:
Addressed in https://github.com/hpc-carpentry/hpc-intro/pull/113/commits/f4ca9d0689e31d11a86a77e7cc7671ce627e88da